app/views/errors/incorrect_state/auth_success.njk

```diff
@@ -1,18 +1,18 @@
 {% extends "../../layout.njk" %}
 
 {% block pageTitle %}
-Your payment is in progress - GOV.UK Pay
+  {{ i18n.authorisation.inProgress }} - GOV.UK Pay
```

Comment:
I think I suggested `i18n.commonHeadings.inProgress` to keep this out of `i18n.authorisation`